src/AWS.Deploy.Recipes/RecipeDefinitions/BlazorWasm.recipe

```diff
@@ -109,12 +113,142 @@
+                    "DefaultValue": true,
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AccessLogging.EnableAccessLogging",
+                            "Value": true
+                        }
+                    ]
+                },
+                {
+                    "Id": "ExistingS3LoggingBucket",
+                    "Name": "Logging Bucket",
+                    "Description": "S3 bucket to use for storing access logs",
+                    "Type": "String",
+                    "DefaultValue": true,
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AccessLogging.EnableAccessLogging",
+                            "Value": true
+                        },
+                        {
+                            "Id": "AccessLogging.CreateLoggingS3Bucket",
+                            "Value": false
+                        }
+                    ]
+                },
+                {
+                    "Id": "LoggingS3KeyPrefix",
+                    "Name": "Logging S3 Key Prefix",
+                    "Description": "Optional S3 key prefix to store access logs (e.g. app-name/)",
+                    "Type": "String",
+                    "DefaultValue": null,
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AccessLogging.EnableAccessLogging",
+                            "Value": true
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Id": "PriceClass",
+            "Name": "CloudFront Price Class",
+            "Description": "Configure the edge locations that will respond to request for the CloudFront distribution",
+            "Type": "String",
+            "DefaultValue": "PRICE_CLASS_ALL",
+            "AllowedValues": [ "PRICE_CLASS_100", "PRICE_CLASS_200", "PRICE_CLASS_ALL" ],
+            "ValueMapping": {
+                "PRICE_CLASS_100": "North America and Europe edge locations",
+                "PRICE_CLASS_200": "North America, Europe, Asia, Middle East, and Africa edge locations",
+                "PRICE_CLASS_ALL": "All edge locations (best performance)"
+            },
+            "AdvancedSetting": true,
+            "Updatable": true
+        },
+        {
+            "Id": "EnableIpv6",
+            "Name": "Enable IPv6",
+            "Description": "Control if IPv6 should be enabled for the CloudFront distribution",
+            "Type": "Bool",
+            "DefaultValue": true,
+            "AdvancedSetting": true,
+            "Updatable": true
+        },
+        {
+            "Id": "MaxHttpVersion",
```

Comment:
Not sure why is this called MaxHttpVersion? Can this be `HttpVersion` only?

Reply:
I didn't like the name `HttpVersion` used in CloudFormation. Because if the value is set to HTTP 2 then it means the user can connect with either 2.0 or 1.1 but of the value is HTTP 1.1 then only a 1.1 connection can be made. By saying `HttpVersion` in our property I don't think we are conveying the true meaning of the property for users to understand.